docs/ui/interactions/fragments/web/chatbot/migration.md

```diff
@@ -0,0 +1,115 @@
+## Migration
+
+To migrate from the legacy chatbot component to the latest chatbot component, use the steps below:
+
+### Usage
+
+<docs-filter framework="react">
+
+```diff
+- import { ChatBot } from 'aws-amplify-react';
++ import { AmplifyChatbot } from '@aws-amplify/ui-react';
```

Comment:
Can you look to make sure the diffs are pointing to the preview as well for all below

Reply:
nice catch, thank you

Reply:
I'll append a comment to each of `/ui-<framework>` then.

Reply:
done!